plone/app/contenttypes/browser/collection.py

```diff
@@ -76,6 +76,7 @@ def album_folders(self):
         """
         return self._album_results['folders']
 
+    @property
```

Comment:
This breaks for anyone who currently calls `view.tabular_fields()` in custom code.

Reply:
Right !